cfgov/jinja2/v1/events/_macros.html

```diff
@@ -130,8 +130,20 @@
 {% macro event_venue(event, event_state) %}
     {%- set city =  event.venue_city + ', ' + event.venue_state | default('Washington, DC') -%}
     <section class="event-venue">
-        <header class="event-venue_header">
-            <h2 class="event-venue_heading">{{ city }}</h2>
+        <div class="event-venue_details">
+            <header class="event-venue_header">
+                {% if ( event_state == 'present' ) %}
+                <a class="btn
+                          event-venue_live-btn"
+                   href="/">
```

Comment:
Add `TODO` here too about updating the link, or will this stay as `/`? Also, maybe mention what has to happen for it to be updated.